Description: update schema to have a standard Sans font
 The greeter defaults to a GNOME based GTK+ theme but with a specific
 Ubuntu only font. This patch changes the Ubuntu only font to Sans
 font found in Debian and all Debian derived distributions
Author: David Mohammed <fossfreedom@ubuntu.com>
Origin: other
Forwarded: not-needed
Last-Update: 2017-05-15

--- a/data/x.dm.slick-greeter.gschema.xml
+++ b/data/x.dm.slick-greeter.gschema.xml
@@ -58,7 +58,7 @@
       <summary>Icon theme to use</summary>
     </key>
     <key name="font-name" type="s">
-      <default>'Ubuntu 11'</default>
+      <default>'Sans 11'</default>
       <summary>Font to use</summary>
     </key>
     <key name="xft-antialias" type="b">
